### DISCUSSION TITLE
Am I the only one?

### DISCUSSION THREAD

**Discussion by hittjwX** [brianpetro/obsidian-smart-connections#837] [General]
Over the last couple of updates, Smart Connection stopped working. I've tried troubleshooting, deleting folders, and uninstalling/reinstalling, but it doesn't work.

![image](https://github.com/user-attachments/assets/c2efb802-ee2a-4af2-ba60-cd6ed5c24d38)

It's been hung like this for more than an hour. I'm a big fan of the plugin. Is anyone having problems lately? How could I roll back?

**Comment by hittjwX** [comment]
I'm making some progress. Changing the configuration to GPU v3 gets me a longer and faster rebuild. However, things still hang eventually.

![image](https://github.com/user-attachments/assets/b7270c2e-4486-4683-b6f3-987081d70e88)

I had a brief period yesterday when the chat worked well. Then `View` started not changing between documents or being blank. V2.2.81.

**Comment by hittjwX** [comment]
It can hang like this for hours. I've reinstalled, deleted the `.smart-env/multi`, manually rebuilt, and now upgraded. What else can I try?

![image](https://github.com/user-attachments/assets/847d6bc5-90bf-45a8-a135-b995f1debc3e)
 
At this point, it's in the way of getting work done. Here's my configuration screen as it is at v2.2.82.

![image](https://github.com/user-attachments/assets/07b00f90-e0de-40bc-8c32-e96a3870481f)

**Reply by BertTheDragon** [reply]
I am experiencing the same problem! Did you get it resolved?

**Reply by hittjwX** [reply]
No, I'm taking a break from Smart Connections. The problem was getting in the way of deadlines here. I'll revisit it in December. I tried everything.

**Comment by brianpetro** [comment]
Closing because there's no obvious issue here.

Screenshots should be of clear errors or console logs to help with debugging 🌴

**Reply by brianpetro** [reply]
While these instructions are specific to the supporter private chat, they can be used as good guidelines on how to participate in our open-source community 

![Screenshot_20241027-084008](https://github.com/user-attachments/assets/c9a0afb3-e0b4-4b63-b806-fe3acb26a31c)
![Screenshot_20241027-084026](https://github.com/user-attachments/assets/49e3ea6b-bc1f-4f88-9ee0-29121a08aa2a)
🌴
